client/src/js/field/googlemapsfield.js

```diff
@@ -25,11 +25,13 @@ const fillAddress = () => {
 };
 
 const mountMapsField = (mapsfield) => {
-  options = Object.assign(options, window.customisations);
-  autoComplete = new google.maps.places.Autocomplete(mapsfield, options);
-  autoComplete.addListener('place_changed', fillAddress);
+  const fieldName = mapsfield.name;
+  options = Object.assign(options, window[fieldName + 'Customisations']);
+  autoComplete[fieldName] = new google.maps.places.Autocomplete(mapsfield, options);
+  autoComplete[fieldName].addListener('place_changed', function(){
```

Comment:
Shouldn't be a function but a direct call. Using a function (by the way, this format is _not_ passing JS linting either) for the callback is like calling your cat with "Hey cat named cat, get over here", instead of just "Cat, get over here"